pages/glossary/explicit-section-of-content.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Explicit section of content
+key: explicit-section-of-content
+unambiguous: true
+objective: true
+---
+
+An _explicit section of content_ is a subtree of the [flat tree][], rooted at an element with a [semantic role][] that inherits either from the [`landmark`][landmark] or [`document`][document] roles, except if it is the only child of an element with a [semantic role][] that inherits either from the [`landmark`][landmark] or [`document`][document] roles.
```

Comment:
Should this include application roles? Also, I'm not sure here, does this include role=document itself? If not, why not?

Reply:
I've improved the formulation to make it clear that the `document` role is included.

I've currently left out the `application` role as I'm not entirely sure about it (and how it is used), and because it's not considered at all in the outline algorithm. But it's not a strong feeling and I'll add it if experts think it's needed.